syncapi/storage/postgres/current_room_state_table.go

```diff
@@ -112,7 +112,7 @@ const selectEventsWithEventIDsSQL = "" +
 const selectSharedUsersSQL = "" +
 	"SELECT state_key FROM syncapi_current_room_state WHERE room_id = ANY(" +
 	"	SELECT room_id FROM syncapi_current_room_state WHERE state_key = $1 AND membership='join'" +
-	") AND state_key = ANY($2) AND membership='join';"
+	") AND state_key = ANY($2) AND membership IN ('join', 'invite');"
```

Comment:
Shouldn't this be `membership = ANY('join', 'invite')` for PostgreSQL?

Reply:
`membership = ANY(ARRAY['join', 'invite'])`, I think `IN` is easier to read.

Reply:
If it works then ship it!